### DISCUSSION TITLE
Default value for new bookmarks

### DISCUSSION THREAD

**Discussion by mrburns-42** [LeoKlaus/plappa#96] [Feature Requests]
When I bookmark something, I usually don't know what to write as a comment. So I put a finger on the keyboard and swipe over it to create some words. Then I save it ... that's what I do in Plappa, as well as in some other apps.

The ABS app always provides a default value, the date with the time:

![IMG_6757](https://github.com/LeoKlaus/plappa/assets/786982/55f5cd1d-d312-4fd1-94f9-da86bdf5aba4)

I would like it if this function was also available in Plappa. My bookmarks would then be much more useful than the way I do it now. If necessary, you could also give the user the option of using a slider in the app settings to select whether they want the default value to be an empty field or the date and time.

**Comment by LeoKlaus** [comment]
I just pushed an update to TestFlight to address this.
I changed the default behavior of the bookmark alert; if no text is entered, it will add the bookmark using the current date and time.

There's also a new toggle in the settings to enable "quick bookmarks". If this is on, tapping the bookmark icon will instantly add a new bookmark with the current date and time as name, without showing any prompt.

**Reply by mrburns-42** [reply]
Thank you very much, that helps immensely. After much trial and error, I have noticed two things:

Would it perhaps be useful for a short confirmation to be discreetly displayed after adding such a bookmark? - Perhaps a green tick or the text “The bookmark [Sting] has been added.”

For new users it is not clear that such a bookmark can be set in this way. But I also don't know how this could be done elegantly. As an idea, I would have the option in the new bookmark menu item to set the default value using the slider and this would then always be in the field. Alternatively, I could imagine that the entry would be the default value in the field. In this case, however, it is only a background value and is immediately removed as soon as you enter something in the bookmark name field.

Please don't get me wrong, you've done a great job. These are just ideas to perhaps improve it, if you wish.
